sdcm/logcollector.py

```diff
@@ -624,8 +624,10 @@ def collect_logs_per_node(node):
                 except Exception as details:  # pylint: disable=unused-variable, broad-except
                     LOGGER.error("Error occured during collecting on host: %s\n%s", node.name, details)
 
+        LOGGER.info("Nodes list %s", [node.name for node in self.nodes])
```

Comment:
this will show an empty list when there are no nodes... 
It should be debug log level or should be inside the "if" below

Reply:
change to debug